### ISSUE TITLE
Code query

### ISSUE THREAD

**Issue by Tom1123456** [keziyi-buct/TRNN#2]
Hello, I would like to ask about the scatter plots in your TRN1-1, do they seem to me to be all from the training set

**Comment by keziyi-buct** [comment]
> Hello, I would like to ask about the scatter plots in your TRN1-1, do they seem to me to be all from the training set
The code provides an example of plotting a scatterplot which is from the training set. You can also write this code in the test part of the model in experiment. By doing this you can plot the scatterplot of the test set. Just adjust it to your needs, the plotting function is encapsulated in def.

**Comment by Tom1123456** [comment]
Thank you very much for your reply, that is, I saw that the scatterplot of the code you published used the data of the training set, and then I have this problem. Another question is whether the scatterplot in your paper and the data in Table 3 compared with others are from the test set

**Comment by Tom1123456** [comment]
Thank you very much for your reply, that is, I saw that the scatterplot of the code you published used the data from the training set, and then I have this problem. Another question is whether the scatterplot in your paper and the results compared with others in Table 3 used data from the test set

**Comment by Tom1123456** [comment]
> > 您好，我想问一下您的 TRN1-1 中的散点图，在我看来它们是否都来自训练集代码提供了一个绘制来自训练集的散点图的示例。你也可以在 experiment 中模型的 test 部分编写此代码。通过执行此作，您可以绘制测试集的散点图。只需根据您的需要进行调整，绘图功能就封装在 def 中。

**Comment by Tom1123456** [comment]
> > 您好，我想问一下您的 TRN1-1 中的散点图，在我看来它们是否都来自训练集代码提供了一个绘制来自训练集的散点图的示例。你也可以在 experiment 中模型的 test 部分编写此代码。通过执行此作，您可以绘制测试集的散点图。只需根据您的需要进行调整，绘图功能就封装在 def 中。
Thank you very much for your reply, that is, I saw that the scatterplot of the code you published used the data from the training set, and then I have this problem. Another question is whether the scatterplot in your paper and the results compared with others in Table 3 used data from the test set
